media-radio/ax25-apps/ax25-apps-0.0.8_rc5-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+MY_P="${P/_/-}"
+
+DESCRIPTION="Basic AX.25 (Amateur Radio) user tools, additional daemons"
+HOMEPAGE="https://packet-radio.net/ax-25/"
+SRC_URI="https://ham.packet-radio.net/packet/ax25/ax25-apps/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="
+	>=dev-libs/libax25-0.0.12_rc2:=
+	sys-libs/ncurses:=
+	virtual/pkgconfig
+	!dev-ruby/listen
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_prepare() {
+	# fix missing prototype for malloc
+	sed -i -e "/^#include /i #include <stdlib.h>" ax25ipd/routing.c || die
+	eapply "${FILESDIR}"/${PN}-0.0.8_rc5-tinfo.patch
+	eapply_user
+	eautoreconf
+}
+
+src_install() {
+	default
+
+	newinitd "${FILESDIR}"/ax25ipd.rc ax25ipd
+	newinitd "${FILESDIR}"/ax25mond.rc ax25mond
+	newinitd "${FILESDIR}"/ax25rtd.rc ax25rtd
+
+	rm -r "${ED}"/usr/share/doc/${PN}* || die
```

Comment:
That's weird, are there more than one dir? Can you show the content of `/usr/share/doc/` in the build image dir? That would be an issue by itself.

Reply:
As a quick hacky debug thing, you could change that to `rm -v -r` so that you see what it's removing in the build.log :)

Reply:
Ah, okay, that would be `${PF}` instead. https://devmanual.gentoo.org/ebuild-writing/variables/index.html

The *proper* way to fix this though would be to look into the build system and make a patch disabling the creation/installation of these doc archives in the first place.